web/styles/search.css

```diff
@@ -215,12 +215,37 @@
         }
     }
 
-    /* for both input container and search suggestions */
-    .user-pill-container {
+    #searchbox-input-container .user-pill-container,
+    .typeahead-menu .user-pill-container {
+        border: 1px solid transparent;
+
         > .pill-label {
             min-width: fit-content;
             white-space: nowrap;
             width: fit-content;
+            /* Replaced by the 5px gap. */
+            margin-right: 0;
+        }
+
+        .pill {
+            /* Reduce user pill height by 2px to
+            preserve an apparent border around
+            them even under `box-sizing: border-box`
+            in the area. */
+            height: calc(var(--height-input-pill) - 2px);
```

Comment:
Isn't this the same `calc()` as is done on `--length-search-pill-image`? Why wouldn't we use that variable here, or perhaps rename it to something more generic?

Reply:
I think you forgot to actually use `length-user-pill-height` here though?